src/app/home-page/home-logged-out/home.component.scss

```diff
@@ -150,10 +150,6 @@ span {
   }
 }
 
-.youtube {
```

Comment:
Is this https://github.com/dockstore/dockstore/issues/2693 ?

Reply:
ya, forgot to mention